plugins/User/config.py

```diff
@@ -43,8 +43,9 @@ def configure(advanced):
 
 
 User = conf.registerPlugin('User')
-# This is where your configuration variables (if any) should go.  For example:
-# conf.registerGlobalValue(User, 'someConfigVariableName',
-#     registry.Boolean(False, """Help for someConfigVariableName."""))
+conf.registerGlobalValue(User, 'listInPrivate',
+    registry.Boolean(True, _("""Determines whether the output of 'user list'
```

Comment:
I would prefer the default to be False, so it follows the generic behavior by default